README.md

```diff
@@ -30,6 +30,6 @@
 - Use `_` between a job name and the [job template variables](https://docs.openstack.org/infra/jenkins-job-builder/definition.html#job-template)
 - Use standard capitalization rules, template variables can be an exception to this
 - Examples:
-  - `RPC-AIO_{series}-{context}-{ztrigger}`
-  - `RPC-AIO_master-swift-periodic`
-  - `Merge-Trigger-JJB`
\ No newline at end of file
+  - `RPC-AIO_{series}-{image}-{scenario}-{action}-{ztrigger}`
+  - `RPC-AIO_master-xenial-swift-deploy-periodic`
```

Comment:
It may make sense, then to have a set of upgrade jobs that specifically cater to upgrades and not deployments. This will end up duplicating a lot in the current job structure, which is not great and I'd prefer to avoid it... Once we've cleared all the logic from the gating repo and put it into the rpc-openstack repositories then I think it'll be easier. But in the short term the most maintainable solution appears to be bastardising the series. :/